src/operator/pooling.cc

```diff
@@ -85,6 +85,10 @@ DMLC_REGISTER_PARAMETER(PoolingParam);
 MXNET_REGISTER_OP_PROPERTY(Pooling, PoolingProp)
 .describe(R"code(Perform pooling on the input.
 
+The shapes for 1-D pooling are
+- **data**: *(batch_size, channel, width)*,
```

Comment:
add a blank line before `-`, otherwise sphnix fails to recognize it as a list, see http://mxnet.io/api/python/symbol.html#mxnet.symbol.Pooling